src/js/components/hints/symbol-hint.tsx

```diff
@@ -1,38 +1,40 @@
-'use strict';
-const getIn = require('../../util/immutable-utils').getIn;
-const lookup = require('../../ctrl').lookup;
-const updateMarkProperty = require('../../actions/markActions').updateMarkProperty;
-
 import * as React from 'react';
 import {connect} from 'react-redux';
 import { Dispatch } from 'redux';
+import {updateMarkProperty} from '../../actions/markActions';
 import {State} from '../../store';
-interface SymbolHintsProps {
-    selectedId: number,
-    updateProperty: (id: number, property: any, value: any) => void // TODO: fix 'any' type
+
+const lookup = require('../../ctrl').lookup; // TODO(jzong) ??? what is this? doesn't exist?
```

Comment:
`Hints` were the newest most experimental part of the `lyra2` codebase. I wouldn't be surprised if a bunch of this code is super incomplete.